tests/containers/helm.pm

```diff
@@ -44,18 +44,22 @@ sub run {
     else {
         install_kubectl();
 
+        set_var("PUBLIC_CLOUD_CREDENTIALS_URL_BASE", get_required_var("PUBLIC_CLOUD_CREDENTIALS_URL")) unless get_var("PUBLIC_CLOUD_CREDENTIALS_URL_BASE");
```

Comment:
This might confuse us in the future.

Let's assume you work on a test case and forgot to set the variables. openQA will in this case complain about the missing variable `PUBLIC_CLOUD_CREDENTIALS_URL`, however it should complain about the missing `PUBLIC_CLOUD_CREDENTIALS_URL_BASE` variable instead, as this is the primary reference.

Is there any good reason for overtaking one variable with another one? This increases the risk of a confused poor QA engineer in the future.

Reply:
I would simply remove this section unless necessary. Overtaking variables adds unnecessary redundancy which makes our lives often more difficult for no real advantage IMHO.

Reply:
There has been a [long discussion](https://suse.slack.com/archives/C02CGKBCGT1/p1654855363080599) about this, even 2 closed PRs. I agree with you, I don't like this hack, and we need to agree on a solution for these cases and make it generic. There was somewhere proposed to re-use the idea of namespaces we have in Vault. But we need agreement before merging this.